packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -1052,7 +1052,7 @@ export function buildColumns(
           fieldName: propName,
           minWidth: MIN_COLUMN_WIDTH,
           maxWidth: 300,
-          isCollapsable: !!columns.length,
+          isCollapsible: !!columns.length,
```

Comment:
You should continue setting isCollapsable until v7--there's a very good chance something downstream is reading it.